src/Text/Pretty/Simple/Internal/OutputPrinter.hs

```diff
@@ -188,9 +187,9 @@ hCheckTTY h options = liftIO $ conv <$> tty
 -- | Parse a string, and generate an intermediate representation,
 -- suitable for passing to any /prettyprinter/ backend.
 -- Used by 'Simple.pString' etc.
-layoutString :: OutputOptions -> String -> SimpleDocStream AnsiStyle
+layoutString :: OutputOptions -> String -> SimpleDocStream Style
 layoutString opts =
```

Comment:
Perhaps we should actually document this functionality properly, and expose this function from a non-Internal module?